scripts/new_project.kts

```diff
@@ -1,25 +1,68 @@
+#!/usr/bin/env kscript
+
+@file:MavenRepository("central", "https://repo.maven.apache.org/maven2/")
+@file:DependsOn("org.jetbrains.kotlinx:kotlinx-coroutines-core:1.6.1")
```

Comment:
Could we also sort these alphabetically? (`@file:DependsOn` in front of `@file:MavenRepository`) 🙇